ui/app/components/ui/export-text-container/export-text-container.component.js

```diff
@@ -5,25 +5,33 @@ import { exportAsFile } from '../../../helpers/utils/util'
 import Copy from '../icon/copy-icon.component'
 
 class ExportTextContainer extends Component {
+  state = {
+    copied: false,
+  }
+
   render () {
     const { text = '' } = this.props
     const { t } = this.context
 
     return (
       <div className="export-text-container">
         <div className="export-text-container__text-container">
-          <div className="export-text-container__text notranslate">
-            {text}
-          </div>
+          <div className="export-text-container__text notranslate">{text}</div>
         </div>
         <div className="export-text-container__buttons-container">
           <div
             className="export-text-container__button export-text-container__button--copy"
-            onClick={() => copyToClipboard(text)}
+            onClick={() => {
+              this.setState({ copied: true })
+              setTimeout(() => this.setState({ copied: false }), 3000)
```

Comment:
This will throw an error in the console if this component is unmounted when this timer resolves. It's a harmless error in this case, but still nice to avoid. It could be avoided by saving the timeout ID, and clearing the timeout on unmount.

Reply:
The same applies to the similar change made to the `ViewContact` component as well